README.md

```diff
@@ -18,7 +21,26 @@ No. This is a joke.
 
 ### Y U HATE RUST SO MUCH?
 
-I don't, actually. I believe that those who spend their time asking people to rewrite their projects are probably not themselves active Rust developers, as those active devs are probably busy [writing memory-safe code](https://trac.torproject.org/projects/tor/ticket/11331).
+I don't, actually. I believe that those who spend their time asking people to rewrite their projects are probably not themselves active Rust developers, as those active devs are probably busy [writing memory-safe code][memory-safe].
 
 ### R U OFFENDING ME?
 ![](rust.png)
+
+## Building
+This readme is written in Rust to further rebut arguments that Rust has low readability.
```

Comment:
Who is arguing this? I'd prefer you not use my repo to pick fights with even more random people on the internet.

Reply:
k, could delete this